server/color.ts

```diff
@@ -1,4 +1,7 @@
 export class Color {
+    public static WHITE = new Color(255, 255, 255);
```

Comment:
Why not use this?
https://github.com/Pecamo/orbital/blob/f2f3a64756bd88fc6568367a312d6db133fc63e1/server/htmlColors.ts#L147